docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+
+**Mesosphere DCOS**, [mesosphere.com](http://mesosphere.com/product/): provides a highly-portable, operations-driven environment that is fault-tolerant, scalable, elastic and supports multi-service deployments. It seamlessly supports both Docker (isolation-level) and Kubernetes (service-level) in an enterprise-grade, [multi-services](http://www.slideshare.net/timothysc/apache-coneu) environment.
+
+**Multi-service**. The ability of a platform to run multiple operational and analytical workloads on the same cluster, resulting in resource-efficiency, while providing elasticity and isolation capabilities. For example, Web servers (like nginx), App servers (such as node.js) using relational databases (e.g., MySQL) for the operational part can be used together with Hadoop/HDFS, Spark, Kafka, etc. for the analytics part, in one cluster.
+
+## 2. Use Cases
+
+Kubernetes on DCOS delivers the most value for those use cases that share one or more of the following characteristics:
+
+* Production systems employing very large clusters, in the 1000s of nodes.
+* Applications that run on-premise, in a private datacenter.
+* The need for a multi-service setup, on the same cluster, is a given.
+* A requirement to easily migrate parts or all of the distributed application from a private cloud to a public cloud setup, including Amazon Web Services, Google Compute and Microsoft Azure.
+* Run apps in a highly-automated environment in order to reduce operational burden.
+* Reliability and fault-tolerance for all workloads.
+* Enterprise-grade security.
+* Elastic scaling of resources between Kubernetes and other apps.
+
+In the following we will have a look at two real-world case studies in greater detail.
+
+### 2.1 Case study 1: Financial Industry
+
+A leading, multi-national investment bank has a number of applications—from fraud detection to trading and marketing—running on the Mesosphere DCOS and decides to leverage Kubernetes on DCOS in order to better manage their workloads. The majority of the datasets reside on-premise due to various legislative requirements (Sarbanes-Oxley, Dodd-Frank, etc.). However, especially for targeted marketing and sentiment analysis use cases, said investment bank has a 500 node cluster in a public cloud, in addition to their on-premise machines. Kubernetes on DCOS in this scenario is used to enable them to build and run multiple services—stream processing of public news and social media feeds, machine learning to detect fraudulent behavior, recommendations for customers and reaching out to prospects—on both  their private and public cloud machines.
+
+### 2.2 Case study 2: Gaming
```

Comment:
Nit: "Online games" might be better, as the first definition of "gaming" is "gambling"